src/countOccurrences.js

```diff
@@ -18,7 +18,16 @@
  *
  */
 function countOccurrences(phrase, letter) {
-  // write code here
+  const phraseArr = phrase.toLowerCase().split('');
```

Comment:
You do not need an array
```suggestion
  const lowerCasePhrase = phrase.toLowerCase();
```